requirements.in

```diff
@@ -1,11 +1,14 @@
-cookiecutter~=1.6
+urllib3~=1.24.2  # https://nvd.nist.gov/vuln/detail/CVE-2019-11324
```

Comment:
whz do you select urllib3 as always in 1.24 branch and jinja2 with >= ? is there a rationale?